pkg/bookmarks/bookmarks.go

```diff
@@ -9,20 +9,37 @@ import (
 	"github.com/BurntSushi/toml"
 	"github.com/mitchellh/go-homedir"
 
+	"github.com/sosedoff/pgweb/pkg/command"
 	"github.com/sosedoff/pgweb/pkg/shared"
 )
 
 type Bookmark struct {
 	Url      string         `json:"url"`      // Postgres connection URL
 	Host     string         `json:"host"`     // Server hostname
-	Port     string         `json:"port"`     // Server port
+	Port     int            `json:"port"`     // Server port
 	User     string         `json:"user"`     // Database user
 	Password string         `json:"password"` // User password
 	Database string         `json:"database"` // Database name
 	Ssl      string         `json:"ssl"`      // Connection SSL mode
 	Ssh      shared.SSHInfo `json:"ssh"`      // SSH tunnel config
 }
 
+func (b Bookmark) SSHInfoIsEmpty() bool {
+	return b.Ssh.User == "" && b.Ssh.Host == "" && b.Ssh.Port == ""
+}
+
+func (b Bookmark) ConvertToOptions() (command.Options, error) {
```

Comment:
you're right. no point in returning error here. forgot to remove the error when i changed port from string to int